### PR TITLE
Only grab sources and galaxies upon request

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR changes the GcnEventPage to only grab sources and galaxies upon request.

### CODE REVIEW

**Review by profjsb — COMMENTED** [review]
Where does the `upon request` happen on the page now, since it's not automatically loaded?

**Comment by mcoughlin** [comment]
@profjsb I leave it to the GcnSelectionForm to take care of.

**Review by Theodlz — APPROVED** [review]
All good!

File: static/js/components/GcnSelectionForm.jsx [hunk 1]
```diff
@@ -81,12 +81,15 @@ const GcnSelectionForm = ({ gcnEvent, setSelectedLocalizationName }) => {
...
   );
 
-  const defaultStartDate = dayjs(gcnEvent?.dateobs).format(
-    "YYYY-MM-DDTHH:mm:ssZ"
-  );
-  const defaultEndDate = dayjs(gcnEvent?.dateobs)
+  const defaultStartDate = dayjs
+    .utc(gcnEvent?.dateobs)
+    .utc()
+    .format("YYYY-MM-DD HH:mm:ss");
```
Comment (guynir42) [inline static/js/components/GcnSelectionForm.jsx:87]: This double `utc()` seems wrong but I'm not sure.

File: static/js/components/GcnEventPage.jsx [hunk 2]
```diff
@@ -252,28 +248,6 @@ const GcnEventPage = ({ route }) => {
...
-    fetchGcnEventObservations(route.dateobs);
-  }, [route, dispatch]);
-
-  useEffect(() => {
-    const fetchGcnEventGalaxies = async (dateobs) => {
-      await dispatch(galaxiesActions.fetchGcnEventGalaxies(dateobs));
-    };
-    fetchGcnEventGalaxies(route.dateobs);
-  }, [route, dispatch]);
-
```
Comment (guynir42) [inline static/js/components/GcnEventPage.jsx:276]: This seems important but if you checked it works without all this stuff then it is fine with me.

**Review by guynir42 — APPROVED** [review]

File: static/js/components/GcnSelectionForm.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/GcnSelectionForm.jsx]: It was just unnecessary, yeah.

File: static/js/components/GcnEventPage.jsx (hunk 2 above)
Comment (mcoughlin) [reply static/js/components/GcnEventPage.jsx:276]: Yeah it just means we have to specifically ask for it in the selection form.

**Review by guynir42 — APPROVED** [review]

**Pull request merged**